spec/requests/api/v2/green_lanes/subheadings_controller_spec.rb

```diff
@@ -34,6 +34,47 @@
       it { is_expected.to have_http_status(:not_found) }
     end
 
+    context 'when the good nomenclature has applicable measures' do
+      before do
+        measure1 = create(:measure, measure_generating_regulation_id: 'D0000001', measure_type_id: '400')
+        measure2 = create(:measure, measure_generating_regulation_id: 'D0000002', measure_type_id: '500')
+        subheading = create(:subheading, goods_nomenclature_item_id: '1234560000', producline_suffix: '80')
+
+        allow(subheading).to receive(:applicable_measures).and_return [measure1, measure2]
+      end
+
+      it_behaves_like 'a successful jsonapi response'
+    end
```

Comment:
Probably, we do not need this test.

Reply:
I'd rather leave that to test integration of applicable measures in subheading serialiser